android-pay/build.gradle

```diff
@@ -2,6 +2,8 @@ apply plugin: 'com.android.library'
 
 def android_support_version = '25.3.1'
 
+version = VERSION_NAME
```

Comment:
Moving the version name into the individual build.gradle files lets me have two different ones (instead of both of them being 4.0.1)